Add cfp to header.

avaldivi opened the pull request.

M0nica commented:
Do you want this Airtable form to replace the system of creating an issue to submit a talk proposal https://github.com/react-ladies/react-ladies-talk-proposals/issues/new/choose? If so, can you update the FAQ so that we have consistent messaging? https://reactladies.com/faq/#how-to-speak


In terms of the audience do you think the word “CFP” in the navigation is familiar enough? Or should we use a different word to describe where someone can submit a talk? Maybe we could combine the get involved, FAQ, and info to submit a talk on a top level “getting involved” page https://reactladies.com/getting-involve


I’m open to combine some of the pages that are current in the top level navigation and am not 100% tied to the FAQ remaining in the top level nav if space is an issue as long as the FAQ has multiple links to it throughout the site. 

What are your thoughts?

avaldivi commented:
> Do you want this Airtable form to replace the system of creating an issue to submit a talk proposal https://github.com/react-ladies/react-ladies-talk-proposals/issues/new/choose? If so, can you update the FAQ so that we have consistent messaging? https://reactladies.com/faq/#how-to-speak
> 
> In terms of the audience do you think the word “CFP” in the navigation is familiar enough? Or should we use a different word to describe where someone can submit a talk? Maybe we could combine the get involved, FAQ, and info to submit a talk on a top level “getting involved” page https://reactladies.com/getting-involve
> 
> I’m open to combine some of the pages that are current in the top level navigation and am not 100% tied to the FAQ remaining in the top level nav if space is an issue as long as the FAQ has multiple links to it throughout the site.
> 
> What are your thoughts?

"CFP" might seem to serious or even professional and might scare away people new to speaking. I updated to "Submit a talk".
I removed FAQ from the header to the footer and updated the FAQs. We should keeps the FAQs page, as I see the page growing soon.
